create_coda_files/create_coda_files.py

```diff
@@ -58,32 +64,48 @@ def _open_scheme(filepath):
     traced_json_output_path = args.traced_json_output_path
     coda_output_path = args.coda_output_path
     coding_scheme_path = args.coding_scheme_path
-    if args.auto_cleaner not in AUTO_CLEANERS.keys():
-        auto_cleaner = args.auto_cleaner
-        auto_cleaner = AUTO_CLEANERS[auto_cleaner]
+    if args.has_yes_no == 'True':
+        has_yes_no = True
+    elif args.has_yes_no == 'False':
+        has_yes_no = False
+    else:
+        raise Exception('<has-yes-no> only takes the strings "True" or "False"')
+    if args.auto_cleaner in AUTO_CLEANERS.keys():
+        auto_cleaner = AUTO_CLEANERS[args.auto_cleaner]
     else:
+        print('There is no matching cleaner for: {}'.format(args.auto_cleaner))
```

Comment:
Shouldn't this be a warning (or an error)?

Reply:
It's a warning, but not an error. This can be run without a cleaner. I'm wondering if there should an explicit option for when there's no cleaner like None